### PR TITLE
Implement internationalization `PapercupsIntl()` + some fixes

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hi @aguilaair!

This PR implements a couple of things:

1. The new `PapercupsIntl()` class that brings translations for all strings I was able to find in the source code. One can now use it as such:
```dart
PaperCupsWidget(
  props: Props(
    accountId: "xxxxxxxx-xxxxxxx-xxxx-xxxxxx",
    translations: PapercupsIntl(
      title: "Welcome 👋"
      subtitle: "How can we help you today?",
      greeting: "We'll make sure to reply in an hour or less",
      // ...
    ),
  ),
)
```

⚠️  **BREAKING CHANGE**

This is a breaking change as I've removed the duplicated properties from both `Props` and `PapercupsWidget`. My rationale is that this could be valid enough for a major release if coupled with the Availability feature for instance. Let me know If you'd rather have it in a minor release so that I add deprecation notices a little bit everywhere. 

2. A [fix](https://github.com/papercups-io/papercups_flutter/commit/39260e9c495fca7916965e0e8d61a67dd8a1af52) for a bug I found when running apps in `profile` or `release` mode: `event.event.toString()` only returns `"PhoenixChannelEvent"` in those modes. So this case was never `true` https://github.com/papercups-io/papercups_flutter/blob/95252be186b383b0cbfad8c059e6dd3bf57793e9/lib/utils/socket/joinConversation.dart#L32-L33
which means that whenever a message was sent from the dashboard to an app, it wouldn't show up in real-time.

3. Replacing the use of Papercups `fullName` for `displayName` as that seems more appropriate in this context.

4. A couple of minor typos corrections here and there.

### CODE REVIEW

**Comment by aguilaair** [comment]
This looks awesome 😎

TYSM, I'll review it ASAP

File: lib/utils/socket/joinConversation.dart [hunk 1]
```diff
@@ -30,7 +30,8 @@ PhoenixChannel? joinConversationAndListen({
           conversation = null;
         } else {
           if (event.event.toString().contains("shout") ||
-              event.event.toString().contains("message:created")) {
+              event.event.toString().contains("message:created") ||
+              event.payload!["type"] == "reply") {
```
Comment (aguilaair) [inline lib/utils/socket/joinConversation.dart:34]: This was a great catch!

**Review by aguilaair — APPROVED** [review]
LGTM!

**Comment by aguilaair** [comment]
I'll merge this now. However, I will not make a release quite yet because there are some kinks in the file upload I want to iron out, specifically want the upload to happen inside of `lib/widgets/chat/attachment.dart` instead of the sendMessage function, this should help concurrency and we can also show more progress details instead of having a snackbar do that.

**Comment by aguilaair** [comment]
closes #67

**Comment by aguilaair** [comment]
Regarding a major version release, I agree that this being a breaking PR it should be released as a major version in order to ensure minimal issues. We can (maybe?) pair it with attachments, although availability should be able to be implemented now too, I'll eventually get to it but right now I have no time to work on it. Hopefully will get a chance in the next month or so, or maybe you wanna take a stab at it? Let me know and I'll assign it, @CharlesMangwa.

**Comment by CharlesMangwa** [comment]
Thanks for the quick merge @aguilaair! And yes, I can look into Availability. I'd just need a little bit of guidance as I'm not really familiar with the API and even after looking in the [Presence docs](https://hexdocs.pm/phoenix/Phoenix.Presence.html#content), I didn't quite get what I'm supposed to get 😅 But I guess we can continue this convo in #65.

**Pull request merged**